index.html

```diff
@@ -7,362 +7,403 @@
     <script src="https://code.jquery.com/jquery-3.5.1.min.js" integrity="sha256-9/aliU8dGd2tb6OSsuzixeV4y/faTqgFtohetphbbj0=" crossorigin="anonymous"></script>
```

Comment:
Usually the Django best practice is to put things like jQuery in a ``base.html`` template that the other templates extend, this way it's easier to be consistent across all templates if you add more in the future

Reply:
An example: https://github.com/DDMAL/CantusDB/blob/develop/django/cantusdb_project/templates/base.html

And then this template extends it: https://github.com/DDMAL/CantusDB/blob/develop/django/cantusdb_project/templates/flatpages/default.html

Reply:
Yes, but this is just running as a job in rodan so any template I make here would only be accessible to this one job.

Reply:
Oh ok, that makes sense